src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,17 @@
+package seedu.addressbook.commands;
+
+/* Sorts the list of persons in the address book by their names in alphabetical order */
```

Comment:
This is not the correct format for header comment. Please use the Javadoc format